Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "SkewLinearAlgebra"
 uuid = "5c889d49-8c60-4500-9d10-5d3a22e2f4b9"
 authors = ["smataigne <simonmataigne@gmail.com> and contributors"]
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
We might want to just tag `1.0` since the API has been stable for a while.

Basically, once things are reasonably stable it is better to make package versions 1.x or higher, since that provides [strictly more information in semver](https://discourse.julialang.org/t/encouraging-1-x-versions-for-registered-packages/97171/57).

Reply:
I will leave tagging and releasing v1.0 to one of the maintainers here, if that is okay